llvm/docs/HowToUpdateDebugInfo.rst

```diff
@@ -115,10 +119,12 @@ Examples of transformations for which this rule *does not* apply include:
   single-stepping experience. The rule for
   :ref:`dropping locations<WhenToDropLocation>` should apply here.
 
-* Hoisting identical instructions which appear in several successor blocks into
-  a predecessor block (see ``BranchFolder::HoistCommonCodeInSuccs``). In this
-  case there is no single merged instruction. The rule for
-  :ref:`dropping locations<WhenToDropLocation>` applies.
+* Hoisting/sinking a part of the identical instructions with the same location.
```

Comment:
The sentence above reads a bit broken, it's unclear what "a part of the identical instructions" is.
Maybe something like: "Hoisting/sinking that would make a location reachable when it previously wasn't"

Reply:
Thanks! This is a good suggestion.